uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/absence-periods.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Prefix this file with `leave-absense-admin-*`. This way we know which scenarios are related to L&A